### ISSUE TITLE
running `npx wow-downloader run-wow` fails, file `PowerPointTransformer.exe` not found.

### ISSUE THREAD

**Issue by larryr1** [larryr1/WOW#3]
```
Getting download url for shared WOW.
Downloading shared WOW.
Running transformer.
node:internal/errors:856
  const err = new Error(message);
              ^

Error: Command failed: C:\Users\###\Desktop\WOW\test\src\PowerPointTransformer.exe wow.pptx
The system cannot find the path specified.

    at ChildProcess.exithandler (node:child_process:410:12)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1091:16)
    at ChildProcess._handle.onexit (node:internal/child_process:302:5) {
  code: 1,
  killed: false,
  signal: null,
  cmd: 'C:\\Users\\###\\Desktop\\WOW\\test\\src\\PowerPointTransformer.exe wow.pptx'
}

Node.js v18.7.0
```

**Comment by larryr1** [comment]
Workaround: install the package and run `npm start` in the package directory.

**Comment by larryr1** [comment]
Fixed with commit #7ab6260
